### DISCUSSION TITLE
Duino-Coin Mining on 32 bit boards- RE-ARM-BIGTREETECH SKR 32BIT BOARDS. etc

### DISCUSSION THREAD

**Discussion by stevet616** [revoxhere/duino-coin#1623] [Q&A]
CAN YOU MINE DUINO-COIN OR ANY OTHER CRYPTO ON 32BIT BOARDS LIKE THE RE-ARM IN PATICULAR? 

![WIN_20230222_03_52_25_Pro](https://user-images.githubusercontent.com/125909287/220570177-a4a6b7d9-e57e-4ed0-bab0-038f2ac29a99.jpg)
![WIN_20230222_03_52_47_Pro](https://user-images.githubusercontent.com/125909287/220570181-6ad4bf59-00d4-4d28-bbcc-dcc4ce5e0d97.jpg)

**Comment by tomasthedev** [comment]
Its possible, you need only right program for it. Can I get that board model/link ?

**Reply by stevet616** [reply]
I'm not sure the link you mean, but the board is just a RAMPS RE-ARM 
![20230222_164912](https://user-images.githubusercontent.com/125909287/220767684-8defe5a2-74f2-4a75-9418-df3afe900688.jpg)

![20230222_165205](https://user-images.githubusercontent.com/125909287/220767907-e765636d-c806-4bf2-928b-2c27368df100.jpg)

**Reply by Grantrocks** [reply]
he means where did you buy the board from. So he can get all the info needed to help you out
